src/ray/object_manager/object_manager.cc

```diff
@@ -108,6 +131,21 @@ ray::Status ObjectManager::SubscribeObjDeleted(
   return ray::Status::OK();
 }
 
+void ObjectManager::PutInlineObject(const ObjectID &object_id,
+                                    const std::vector<uint8_t> &inline_object_data,
+                                    const std::string &inline_object_metadata) {
+  // Called when the object is created from inlined data.
```

Comment:
A better place for a comment like this is in the documentation for the method in `object_manager.h`.